Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.py

```diff
@@ -0,0 +1,1526 @@
+                headers = [header.replace(".", replace_symbol) for header in headers]
+                """ Remove dots from events key names"""
+                events_undot = undot(json_data=events)
+                readable_output = tableToMarkdown(
+                    f"{total} events out of "
+                    + str(events["total"])
+                    + f" retrieved for the {term}:",
+                    events_undot,
+                    headers=headers,
+                )
+
+            # If there is no event info to exclude in the parameters nor in the arguments
+            elif (
+                (exclude_info_arg is None)
+                and (not exclude_info_param)
+                and (len(events["items"]) > 0)
+            ):
+                events_undot = undot(json_data=events)
+                readable_output = tableToMarkdown(
+                    f"{total} events out of "
+                    + str(events["total"])
+                    + f" retrieved for the {term}:",
+                    events_undot,
+                )
+
+            # ELSE
+            else:
+                readable_output = tableToMarkdown(
+                    f"{total} events out of "
+                    + str(events["total"])
+                    + f" retrieved for the {term}:",
+                    events["items"],
+                )
+
+            bool_flag = False
+        else:
+            demisto.debug(f"The query is still running with the status {status}")
+
+        time.sleep(int(args.get("interval_in_seconds") or 0))
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="SekoiaXDR.Events.Results",
+        outputs_key_field="search_job_uuid",
+        outputs=events["items"],
+    )
+
+
+def update_status_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    alert_uuid, updated_status, comment = (
+        str(args.get("id")),
+        str(args.get("status")),
+        args.get("comment"),
+    )
+    sekoia_transition = STATUS_TRANSITIONS.get(updated_status)
+    readable_output = "Unknown alert"
+
+    workflow = client.get_workflow_alert(alert_uuid=alert_uuid)
+
+    for action in workflow["actions"]:
+        if action["name"] == sekoia_transition:
+            update = client.update_status_alert(
+                alert_uuid=alert_uuid, action_uuid=action["id"], comment=comment
+            )
+            if update or update == {}:
+                readable_output = (
+                    "### Alert " + alert_uuid + " updated to status: " + updated_status
+                )
+            else:
+                raise DemistoException(
+                    "Failure to update the status of the alert. \
+                    Run the command !sekoia-get-workflow-alert to see the possible\
+                    transitions and review the code."
+                )
+
+    return CommandResults(readable_output=readable_output, outputs=update)
```

Comment:
Yes, it's because `get_workflow_alert` give us exactly the `id` of the action. And maybe if we delete it in the future we will get errors because the `id` will change. So I think we can keep it like that. 

- You can see the `test_data` folder to see the example.